src/client/pages/Embark/index.tsx

```diff
@@ -208,9 +209,9 @@ interface AngelVariables {
 export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
   const history = useHistory()
   const [data, setData] = React.useState<[string, any] | null>(null)
-  const [initialStore, setInitialStore] = React.useState<null | {
-    [key: string]: any
-  }>()
+  const [initialStore, setInitialStore] = React.useState<Record<string, any>>({
+    quoteCartId: props.quoteCartId,
+  })
```

Comment:
@sampettersson @hugokallstrom @hedvigoscar I guess this is sort of what you had in mind as well to communicate the quote cart ID by setting it in the initial Embark store, right?